### PR TITLE
Feature blackquote

### PR DESCRIPTION
**Pull request by FrontendKedra**

### CODE REVIEW

File: system/libs/figa-ui/src/lib/blackquote/blackquote.stories.tsx
Comment (polubis) [inline system/libs/figa-ui/src/lib/blackquote/blackquote.stories.tsx:1]: In every place where you importing type you need to add type alias - import type... 


It speeds up TS builds.

File: system/libs/figa-ui/src/lib/blackquote/blackquote.test.tsx
Comment (polubis) [inline system/libs/figa-ui/src/lib/blackquote/blackquote.test.tsx:1]: Look how other component tests are described.


It should be 

There is option to use blackquote when:

File: system/libs/figa-ui/src/lib/blackquote/blackquote.tsx
Comment (polubis) [inline system/libs/figa-ui/src/lib/blackquote/blackquote.tsx:1]: Import type

**Review by polubis — CHANGES_REQUESTED** [review]
There is one re-export missing in main lib directory, despite that great work - you can improve it or just leave it to me u2y

File: system/libs/figa-ui/src/lib/blackquote/blackquote.stories.tsx
Comment (FrontendKedra) [reply system/libs/figa-ui/src/lib/blackquote/blackquote.stories.tsx:1]: Thank you for valuable information, I've implemented the fixes to imports

**Comment by polubis** [comment]
@FrontendKedra closing your PR and merging via this one ->
https://github.com/polubis/Dream-stack-for-React-dev/pull/102/files

Which have small improvements. Please check in free time.

**Pull request closed without merging**